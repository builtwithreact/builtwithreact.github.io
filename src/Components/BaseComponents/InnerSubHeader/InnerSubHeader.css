.InnerSubHeader {
  background: #1c1d21;
  color: #fff;
  height: 220px;
  text-align: center;
  display: flex;
  justify-content: center;
  align-items: center;
  font-size: 40px;
  line-height: 1; }
  @media screen and (max-width: 767px) {
    .InnerSubHeader {
      height: 100px;
      font-size: 25px; } }
  .InnerSubHeader .InnerSubHeader__Title {
    font-family: Courgette, sans-serif; }
