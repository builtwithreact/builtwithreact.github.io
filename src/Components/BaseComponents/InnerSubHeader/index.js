import React from 'react';
import PropTypes from 'prop-types';
import './InnerSubHeader.css';


const InnerSubHeader = ({ title }) => {
    return (
        <div className="InnerSubHeader">
            <div className="container">
                <h1 className="InnerSubHeader__Title">
                    {title}
                </h1>
            </div>
        </div>
    )
}

InnerSubHeader.propTypes = {
    title: PropTypes.string.isRequired
};

export default InnerSubHeader;