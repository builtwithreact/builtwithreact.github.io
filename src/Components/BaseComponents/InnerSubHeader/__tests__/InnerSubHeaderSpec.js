import React from 'react';
import renderer from 'react-test-renderer';
import InnerSubHeader from './../index';

describe("<InnerSubHeader />", ()=>{
    it('should return correctly', () => {
        let tree = renderer.create(
            <InnerSubHeader title="InnerSubHeader"/>
        ).toJSON();

        expect(tree).toMatchSnapshot()
    })
})