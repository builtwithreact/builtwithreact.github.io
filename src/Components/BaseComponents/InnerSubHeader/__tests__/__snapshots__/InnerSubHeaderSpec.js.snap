// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<InnerSubHeader /> should return correctly 1`] = `
<div
  className="InnerSubHeader"
>
  <div
    className="container"
  >
    <h1
      className="InnerSubHeader__Title"
    >
      InnerSubHeader
    </h1>
  </div>
</div>
`;
